src/uct/ib/base/ib_iface.c

```diff
@@ -380,8 +382,20 @@ static ucs_status_t uct_ib_iface_init_lmc(uct_ib_iface_t *iface,
     return UCS_OK;
 }
 
+static int uct_ib_find_lcs_cpu(ucs_cpu_set_t const *cpu_cores)
```

Comment:
/home/guy/work/ucx.cq/ucx/src/ucs/type/cpu_set.h:71:12: error: 'uct_ib_find_lcs_cpu' defined but not used [-Werror=unused-function]